CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for `ets:update_counter/3` and `ets:update_counter/4`.
 - Added `erlang:+/1`
 - Added `lists:append/1` and `lists:append/2`
+- Support for zero count in `lists:duplicate/2`.
```

Comment:
Nitpicking time: actually this is is a fix that can be moved to the `Fixed` section.